python/lsst/sconsUtils/builders.py

```diff
@@ -364,6 +359,17 @@ def _quote_paths(pathList):
             outConfigFile.write('@INCLUDE_PATH = %s\n' % _quote_paths(docPaths))
         for incFile in incFiles:
             outConfigFile.write('@INCLUDE = %s\n' % _quote_path(incFile))
+
+        # To work in more than one package, tag file's location must be absolute
+        stdlibTag = os.path.join(ProductDir(env, 'sconsUtils'), 'doc/cppreference-doxygen-web.tag.xml')
+        stdlibHtml = 'http://en.cppreference.com/w/'
+        outConfigFile.write('TAGFILES += "%s=%s"\n' % (stdlibTag, stdlibHtml))
```

Comment:
I assume a tag file specification can't use an environment variable? As written this is one more reason we can't relocate a stack.

Reply:
Does the path need to be quoted? (it might have a space in it).

Reply:
If you mean have an environment variable in the config file, I doubt it. The documentation doesn't mention it, and I did check that you can't use a URL.

Can you explain what you mean by "can't relocate a Stack"? The file is present in the repo, so the path is guaranteed to be valid under all conditions under which `ProductDir` will work (whose absence, I think, would break all of `sconsUtils`).

Reply:
Yep. Sorry. This is a build time thing not a run time thing so ignore me.

Reply:
https://stackoverflow.com/questions/3060710/do-doxygen-config-files-support-variables suggests `$(SCONSUTILS_DIR)` might work. No idea if that is true for tag files.

Reply:
Aha, environment variables are mentioned in the Doxygen manual as part of the general format of config files.

So I can remove the code in the first commit, and just use $(SCONSUTILS_DIR) in `base.inc`. 🙂